src/apps/loyalty/server/enroll_loyalty_applicant.ts

```diff
@@ -0,0 +1,46 @@
+import * as request from "request"
+
+interface CollectorProfileData {
+  loyalty_applicant_at: string
+  confirmed_buyer_at: string
+}
+
+export function fetchCollectorProfile(accessToken: string): Promise<CollectorProfileData> {
```

Comment:
I'll be refactoring this to be a middleware since all the routes need it (in addition to the user). Will probably attach it to `req.profile`